src/Lazy.jl

```diff
@@ -147,7 +147,7 @@ next(::List, xs::List) = first(xs), tail(xs)
 Base.show(io::IO, xs::List) =
   foreach(x->print(io,x), ["("] * interpose(xs, " ") * [")"])
 
-function Base.writemime(io::IO, ::MIME"text/plain", xs::List)
+@compat function show(io::IO, ::MIME"text/plain", xs::List)
```

Comment:
Have pinned `Compat` and included `using Compat` at the very beginning. However tests are failing on all three `Julia` versions. But none seems to be related to deprecations.